ParseLoginUI/src/main/java/com/parse/ParseQueryRecyclerViewAdapter.java

```diff
@@ -0,0 +1,611 @@
+package com.parse;
```

Comment:
Yeah I'm waiting until https://github.com/ParsePlatform/ParseUI-Android/pull/65 is merged.